framework/Sources/VGSFramework/UIElements/Text Field/VGSTextField.swift

```diff
@@ -32,13 +32,34 @@ public class VGSTextField: UIView {
         return textField.secureText
     }
     
+    /// Textfield placeholder string
+    public var placeholder: String? {
+        didSet {
+            textField.placeholder = placeholder
+        }
+    }
+    
     /// You can set padding for text and placeholder
     public var padding = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0) {
         didSet {
             textField.padding = padding
         }
     }
     
+    /// The technique to use for aligning the text
+    public var textAlignment: NSTextAlignment = .natural {
+        didSet {
+            textField.textAlignment = textAlignment
+        }
+    }
+    
+    /// Constants that identify the semantic meaning expected for a text-entry area
+    public var textContentType: UITextContentType! {
+        didSet {
+            textField.textContentType = textContentType
+        }
+    }
```

Comment:
Need to more discovery this property. This property has references to Siri -> not secure for us.